commands/channel_e2e_test.go

```diff
@@ -76,7 +76,7 @@ func (s *MmctlE2ETestSuite) TestSearchChannelCmd() {
 
 		err := searchChannelCmdF(c, cmd, []string{s.th.BasicChannel.Name})
 		s.Require().NotNil(err)
-		s.Require().Equal(`: Channel does not exist., `, err.Error())
+		s.Require().Equal(`: app.channel.get_by_name.missing.app_error, `, err.Error())
```

Comment:
Tests are failing because of this change. err is a regular error, not an AppError so there is no Id for it

Reply:
ok added back regular error